scylla/tests/integration/query_result.rs

```diff
@@ -295,3 +296,171 @@ async fn execute_iter_serialization_error() {
         Err(PagerExecutionError::SerializationError(_))
     )
 }
+
+async fn create_session(table_name: &str) -> Session {
+    let session: Session = create_new_session_builder().build().await.unwrap();
+    let ks = unique_keyspace_name();
+
+    let cql_create_ks = format!(
+        "CREATE KEYSPACE IF NOT EXISTS {} WITH REPLICATION =\
+         {{'class' : 'NetworkTopologyStrategy', 'replication_factor' : 1}}",
+        ks
+    );
+    session.ddl(cql_create_ks).await.unwrap();
+    session.use_keyspace(ks, false).await.unwrap();
+
+    let cql_drop_table = format!("DROP TABLE IF EXISTS {}", table_name);
+    session.ddl(cql_drop_table).await.unwrap();
+
+    let cql_create_table = format!(
+        "CREATE TABLE IF NOT EXISTS {} (id int PRIMARY KEY, val int)",
+        table_name,
+    );
+    session.ddl(cql_create_table).await.unwrap();
+
+    session
+}
+
+async fn check_query_unpaged(insert_num: u64, use_prepared_statements: bool) {
+    let table_name = if use_prepared_statements {
+        "execute_unpaged"
+    } else {
+        "query_unpaged"
+    };
+    let session: Session = create_session(table_name).await;
+    session.await_schema_agreement().await.unwrap();
+
```

Comment:
Ditto: no need to await schema agreement

Reply:
Done.